tools/TestImpact/Test-CertifiedShardMap.ps1

```
@@ -0,0 +1,120 @@
+    param([object] $Map, [object] $Certificate, [long] $ExpectedCertificationRunId)
+
+    foreach ($name in 'schemaVersion', 'sha') {
+        if (-not $Map.PSObject.Properties[$name]) { throw "map is missing $name" }
+    }
+    foreach ($name in 'schemaVersion', 'candidateMapRunId', 'candidateMapSha',
+        'auditSourceRunId', 'auditSourceSha', 'certificationRunId', 'escalated',
+        'wouldRun', 'wouldSkip', 'missCount') {
+        if (-not $Certificate.PSObject.Properties[$name]) { throw "certificate is missing $name" }
+    }
+
+    $schema = ConvertTo-RequiredInteger $Certificate.schemaVersion 'certificate schemaVersion' 1
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'ConvertTo-RequiredInteger' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)